src/Resources/contao/classes/Ajax.php

```diff
@@ -155,11 +158,13 @@ public function executePreActions()
 					if ($e->getCode() == 0)
 					{
 						\System::loadLanguageFile('tl_maintenance');
-						echo '<p class="tl_error">' . $GLOBALS['TL_LANG']['tl_maintenance']['notWriteable'] . '</p>';
-						exit; break;
+						throw new ResponseException(
+							new Response('<p class="tl_error">' . $GLOBALS['TL_LANG']['tl_maintenance']['notWriteable'] . '</p>'),
+							$e
```

Comment:
This seems wrong. You are passing the exception object but the `ResponseException` class actually expects an integer value (`$status`).